tests/test_variable.py

```diff
@@ -137,5 +140,30 @@ def test_invalid_value_type(self):
         with self.assertRaises(AssertionError):
             chainer.Variable(1)
 
+    def test_grad_type_check_pass(self):
+        a = chainer.Variable(np.empty((3,), dtype=np.float32))
+        a.grad = np.ndarray((3,), dtype=np.float32)
```

Comment:
Is there any tool that is equivalent to `assert_nothing_raised` in Ruby.
I am wondering if this test case can handle the exception when this assignment raises it.